scripts/deploy-v2.js

```diff
@@ -242,36 +262,41 @@ async function main() {
       NetworkRegistry: registryReceipt.contractAddress || undefined
     });
 
-    await Promise.all([
-      updateSetting("settlerToken", networkToken.contractAddress, "contracts"),
-      updateSetting("network", network.contractAddress, "contracts"),
-      updateSetting("transactionalToken", networkToken.contractAddress, "contracts"),
-      updateSetting("networkRegistry", registryReceipt.contractAddress, "contracts"),
-      updateTokens({
-        name: await networkToken.name(),
-        symbol: await networkToken.symbol(),
-        isTransactional: false,
-        address: networkToken.contractAddress
-      }),
-      updateTokens({
-        name: await networkToken.name(),
-        symbol: await networkToken.symbol(),
-        isTransactional: true,
-        address: networkToken.contractAddress
-      }),
-      updateTokens({
-        name: await rewardToken.name(),
-        symbol: await rewardToken.symbol(),
-        isTransactional: false,
-        address: rewardToken.contractAddress
-      }),
-      updateTokens({
-        name: await bountyTransactional.name(),
-        symbol: await bountyTransactional.symbol(),
-        isTransactional: true,
-        address: bountyTransactional.contractAddress
-      })
-    ]);
+    if (!argv.production) {
+      await Promise.all([
+        updateSetting("settlerToken", networkToken.contractAddress, "contracts"),
+        updateSetting("network", network.contractAddress, "contracts"),
+        updateSetting("transactionalToken", networkToken.contractAddress, "contracts"),
+        updateSetting("networkRegistry", registryReceipt.contractAddress, "contracts"),
+        updateTokens({
+          name: await networkToken.name(),
+          symbol: await networkToken.symbol(),
+          isTransactional: false,
+          address: networkToken.contractAddress
+        }),
+        updateTokens({
+          name: await networkToken.name(),
+          symbol: await networkToken.symbol(),
+          isTransactional: true,
+          address: networkToken.contractAddress
+        }),
+        updateTokens({
+          name: await rewardToken.name(),
+          symbol: await rewardToken.symbol(),
+          isTransactional: false,
+          address: rewardToken.contractAddress
+        }),
+        updateTokens({
+          name: await bountyTransactional.name(),
+          symbol: await bountyTransactional.symbol(),
+          isTransactional: true,
+          address: bountyTransactional.contractAddress
+        })
+      ]);
+    } else {
+      console.log('Skiping Database Save');
```

Comment:
If we're not saving this information to the database, _how_ are we to know these values when the application queries for them ?

Reply:
They are coming from environment variables right ?! At least was what you said.